Header header
float64 Radius
string Direction
